control/rpmrc

```diff
@@ -1,8 +0,0 @@
-macrofiles: /usr/lib/rpm/macros:/usr/lib/rpm/%{_target}/macros:/etc/rpm/macros.specspo:/etc/rpm/macros:/etc/rpm/%{_target}/macros:%(echo $CONFIGDIR)/rpmmacros
-
-buildarchtranslate: athlon: i386
-buildarchtranslate: i686: i386
-buildarchtranslate: i586: i386
-buildarchtranslate: i486: i386
-buildarchtranslate: i386: i386
```

Comment:
Did we really need to remove those translations? Didn't we break anything with that?

Reply:
All these translations are already defined in the default /usr/lib/rpm/rpmrc used by rpmbuild, also having some other translations for 32bit and 64bit are also in this file (osfmach3, geode, pentium4, pentium3, amd64, ia32e,...). So it seems to me better to use this file that is much more up-to-date.
Some distributions such as Fedora are building all their packages using i386 options for all 32bit intel, but use i686 as architecture in the package name. This is done by the /usr/lib/rpm/redhat/rpmrc file. I think it's best to follow this when building a package for such distributions to have the same settings than other packages.
And in this case if forcing i386 is required for any reason, this can be done by passing--rpmrc /usr/lib/rpm/rpmrc to rpmbuild, or by adding a file /etc/rpmrc or ~/.rpmrc with the required single buildarchtranslate: line.
That's why I think in any case we can remove our control/rpmrc file.